### ISSUE TITLE
Allow Warpcast to become a signer?

### ISSUE THREAD

**Issue by tudormunteanu** [stevedylandev/cast-keys#1]
hey,

This is more of an exploratory question than an issue, so happy to be told a clear "no". :)

Would it be possible to generate these keys for an FID which was created outside Warpcast? 

My intention here is to import this FID into Warpcast/Supercast.

I have tried this with [Opencast](https://github.com/stephancill/opencast/) and it didn't work.

**Comment by stevedylandev** [comment]
Hey!! This is actually a great question! Generally you need to authorize the key with a custody wallet app; this is a client which holds your farcaster account mnemonic and can authorize things like signers. If Opencast has an API similar to Warpcast I would love to see if I can get it integrated! cc @stephancill

**Comment by tudormunteanu** [comment]
Right. That's exactly what Opencast does.

I looked into some of the fundamentals and code behind it and managed to make progress. 

The blocker is actually importing things into Warpcast.

**Comment by stevedylandev** [comment]
Yeah I think I've heard similar things, one of those situations where if you choose to spawn outside the gates you'll have a hard time getting back in 😬

Have you tried importing the seed phrase to Warpcast? I haven't tried it with accounts made outside of Warpcast but I have multiple accounts and thats how I switch between them

**Comment by stevedylandev** [comment]
Also curious if this will be a fix? 👀

https://warpcast.com/horsefacts.eth/0x3bae5283

**Comment by tudormunteanu** [comment]
> Also curious if this will be a fix? 👀
> 
> https://warpcast.com/horsefacts.eth/0x3bae5283

This looks promising! Will try it later today and report back. Cheers for sharing!
